### PR TITLE
updating readme with xamarin forms example

### PR DESCRIPTION
**Pull request by phenomx2**
I'm updating the README.md including a little explanation to Cinephile example in Xamarin Forms

**What kind of change does this PR introduce?**
Docs

**What is the current behavior?**
Helps a little bit with [Fix the readme file](https://github.com/reactiveui/ReactiveUI.Samples/issues/25) issue

**What is the new behavior?**

**What might this PR break?**
Nothing

**Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**Other information**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,4 +20,18 @@ We are showing how Commands are being used side by side by MVVMLight, Caliburn.M
 We are showing how Messaging Pattern (AKA EventAggregation) is being used side by side by MVVMLight, Caliburn.Micro and of course ReactiveUI.
```
Comment (glennawatson) [inline README.md]: I would remove the sections above your change for the moment. They are based on the old-old samples.

File: README.md [hunk 2]
```diff
@@ -20,4 +20,18 @@ We are showing how Commands are being used side by side by MVVMLight, Caliburn.M
...
+## Xamarin Forms 
+
+#### Cinephile
+
+Example in Xamarin Forms (Android and iOS) shows you the upcoming movies from [the movie api](https://api.themoviedb.org/3)
+
+Provides examples about: 
+
+1. App's bootstrapping
+2. Viewmodel first routing
```
Comment (glennawatson) [inline README.md]: This section looks good.

File: README.md (hunk 1 above)
Comment (phenomx2) [reply README.md]: Ok, old sections removed

File: README.md (hunk 2 above)
Comment (phenomx2) [reply README.md]: :)

**Pull request merged**